object-manipulation-2.tar/object-manipulation-2/src/pick.js

```diff
@@ -0,0 +1,10 @@
+/* exported pick */
+function pick(source, keys) {
+  const result = {};
+  for (keys in source) {
+    if (keys.includes(keys)) {
+      result[keys] = source[keys];
+    }
+  }
```

Comment:
Just be careful because i think at least on line 4-5 you are mixing up `keys` and `key`

Try this instead:

loop through the `keys` array from your arguments (a traditional loop... ie `for(let i = 0; i < keys.length; i++)`). in your `for` loop, set a local variable `key` that is the current value of `keys[i]`, then do a conditional that checks if `source[key]` is not undefined - if it isn't `result[key] = source[key]`